presentations/2024-10-10_what-is-ai-tom/index.qmd

```diff
@@ -0,0 +1,434 @@
+---
+title: "Computer Vision"
+subtitle: "Is this AI?"
+author: 
+  - "[Tom Jemmett](mailto:thomas.jemmett@nhs.net)"
+date: 2024-10-10
+date-format: "D MMMM YYYY"
+format:
+  revealjs:
+    theme: [default, ../su_presentation.scss]
+    transition: none
+    chalkboard:
+      buttons: false
+    preview-links: auto
+    slide-number: false
+    auto-animate: true
+    reference-location: document
+    footer: |
+      Learn more about [The Strategy Unit](https://www.strategyunitwm.nhs.uk/)
+---
+
+## What is Computer Vision (CV)
+
+::: {.fragment}
+> Computer vision is a field of **computer science** that focuses on enabling **computers to
+> identify and understand objects and people** in **images and videos**.
+:::
+
+::: {.fragment}
+> Like other types of AI, computer vision seeks to **perform and automate tasks** that **replicate
+> human capabilities**.
+:::
+
+::: {.fragment}
+> In this case, computer vision seeks to replicate both the way **humans see**, and
+> the way humans **make sense** of what they see.
+:::
+
+::: footer
+Source: [What is computer vision? Microsoft](https://azure.microsoft.com/en-gb/resources/
+cloud-computing-dictionary/what-is-computer-vision#object-classification)
+:::
+
+# What tasks can CV be used for? {.center .inverse}
+
+---
+
+## Classification
+
+::: {.fragment fragment-index=6}
+*Assign a single label to each image*
+:::
+
+:::: {.columns}
+
+::: {.column width="40%"}
+
+::: {.fragment fragment-index=1}
+![](https://raw.githubusercontent.com/EliSchwartz/imagenet-sample-images/refs/heads/master/n02102177_Welsh_springer_spaniel.JPEG){height=300px}
+:::
+
+::: {.fragment fragment-index=3}
+Dog
+:::
+::: {.fragment fragment-index=4}
+Welsh Spaniel
+:::
+::: {.fragment fragment-index=5}
+Animial in water
```

Comment:
```suggestion
Animal in water
```

Reply:
oh, I thought I had already corrected that spelling mistake! :-) ta!